websites/A/Anime-Sugoi/dist/metadata.json

```diff
@@ -5,13 +5,11 @@
   },
   "service": "Anime-Sugoi",
   "description": {
-    "en": "Anime-Sugoi stream your favorite anime completely for free with thai-sub and thai sound! | This website may requied vpn for access, other country will blocked!",
-    "th": "Anime-Sugoi เว็ปไซต์สตรีมมิ่งอนิเมะที่ทุกคนชื่นชอบ ฟรีเเละมีซับไทย",
-    "ga_IE": "Sruthlaigh Anime-Sugoi an anime is fearr leat go hiomlán saor in aisce le fuaim thai-sub agus Téalainnis! | Féadfaidh an láithreán gréasáin seo vpn a iarraidh ar rochtain, cuirfear bac ar thír eile!",
-    "nl": "Anime-Sugoi streamt je favoriete anime volledig gratis met thai-sub en thai sound! ​ Deze website heeft mogelijk een VPN verbinding nodig voor toegang, andere landen dab thailand worden geblokkeerd!"
```

Comment:
why removed old ones?

Reply:
The description I think that not good enough then I remove it